extensions/src/main/scala/ai/h2o/sparkling/extensions/stacktrace/StackTraceCollector.scala

```diff
@@ -61,16 +60,15 @@ class StackTraceCollector extends AbstractH2OExtension {
 
   private class StackTraceCollectorThread extends Thread("StackTraceCollectorThread") {
     setDaemon(true)
-
+    import scala.collection.JavaConverters._
     override def run(): Unit = {
       while (true) {
         try {
           Log.info("Taking stacktrace at time: " + new Date)
-          val allStackTraces = Thread.getAllStackTraces
-          import scala.collection.JavaConversions._
-          for (e <- allStackTraces.entrySet) {
-            Log.info("Taking stacktrace for thread: " + e.getKey)
-            for (st <- e.getValue) {
+          val allStackTraces = Thread.getAllStackTraces.asScala
+          for (e <- allStackTraces) {
```

Comment:
What about `for ((key, value) <- allStackTraces) {` ?

Reply:
Great idea!